### ISSUE TITLE
Error deleting event: Field 'comment' doesn't have a default value

### ISSUE THREAD

**Issue by ppanero** [MISP/MISP#2180]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | CentOS
| OS version (client)       | 7
| PHP version               | 5.6
| MISP version / git hash   | 2.4.72
| Browser                   | Firefox

### Expected behavior
Event deletion

### Actual behavior
Attributes of the events get deleted but the event does not. Getting:
```
2017-05-11 08:56:18 Error: [PDOException] SQLSTATE[HY000]: General error: 1364 Field 'comment' doesn't have a default value
Request URL: /events/delete/3030
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(470): PDOStatement->execute(Array)
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(436): DboSource->_execute('INSERT INTO `mi...', Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1090): DboSource->execute('INSERT INTO `mi...')
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1940): DboSource->create(Object(EventBlacklist), Array, Array)
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1758): Model->_doSave(Array, Array)
#5 /var/www/MISP/app/Model/Event.php(334): Model->save(Array)
#6 [internal function]: Event->beforeDelete(false)
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/Event/CakeEventManager.php(241): call_user_func_array(Array, Array)
#8 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(2692): CakeEventManager->dispatch(Object(CakeEvent))
#9 /var/www/MISP/app/Model/Event.php(1117): Model->delete('3030', false)
#10 /var/www/MISP/app/Controller/EventsController.php(1444): Event->quickDelete(Array)
#11 [internal function]: EventsController->delete('3030')
#12 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(491): ReflectionMethod->invokeArgs(Object(EventsController), Array)
#13 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#14 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(EventsController), Object(CakeRequest))
#15 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#16 {main}
```
Tested with events with:
- Attributes with no comment
- Attributes with comment
- Both, attributes with and without comment.

Fails in the three cases

If I try to re-import the schema (deleted database, etc.) I get
```
ERROR 1101 (42000) at line 20: BLOB, TEXT, GEOMETRY or JSON column 'comment' can't have a default value
```

**Comment by ppanero** [comment]
Waiting for myself to test 2.4 latest changes

**Comment by ppanero** [comment]
Fixed in version 2.4.73+
